### PR TITLE
[GS-1846] feat(st-table): Includes new field 'filterable' to apply filters

### PR DESCRIPTION
**Pull request by jdiaz-stratio**
### Documentation
- [x] Add the issue in PR description
- [x] Have changelog updated
- [x] You fill the milestone value
- [x] You add some label
- [x] Have example running in demo app
- [ ] Review id on demo is the same of egeo folder

### Code
- [x] Pass Sass lint task
- [ ] Have qaTags

### Tests
- [ ] Have at least 70% tests coverage

### CODE REVIEW

File: projects/egeo/src/lib/st-table/st-table.component.html [hunk 1]
```diff
@@ -10,27 +10,42 @@
...
+            <st-checkbox *ngIf="i == 0 && selectableAll" qaTag="select-all-checkbox" class="st-table__checkbox" name="selectAll"
+                [checked]="selectedAll" (change)="onSelectAll($event)">
+            </st-checkbox>
+            <label class='st-table__header-text' (click)="onChangeOrder(field)">
+               <i *ngIf="isSortedByField(field)" class='st-table__order-arrow'
+                  [ngClass]="getHeaderItemClass(field)">
+               </i>
+               {{field.label}}
+            </label>
+            <st-pop-over *ngIf="isFilterable(field)" class="st-table__popover" qaTag="pop-over-demo" [title]="getTitle(field)"
```
Comment (emgaitan-stratio) [inline projects/egeo/src/lib/st-table/st-table.component.html]: Remove qaTag="pop-over-demo"

File: projects/egeo/src/lib/st-table/st-table.component.ts [hunk 2]
```diff
@@ -115,17 +123,33 @@ export class StTableComponent implements OnInit {
     */
    @Output() selectAll: EventEmitter<boolean> = new EventEmitter();
 
+   @Output() selectedFilters: EventEmitter<StTableHeader[]> = new EventEmitter();
```
Comment (emgaitan-stratio) [inline projects/egeo/src/lib/st-table/st-table.component.ts]: Why is needed a delay?

File: projects/egeo/src/lib/st-table/st-table.component.html [hunk 3]
```diff
@@ -10,27 +10,42 @@
     SPDX-License-Identifier: Apache-2.0.
 
 -->
-<table class='st-table' [ngClass]="tableClass" [attr.id]='qaTag'>
+<table (stParentsScroll)="onParentScroll($event)" class='st-table' [ngClass]="tableClass" [attr.id]='qaTag'>
```
Comment (emgaitan-stratio) [inline projects/egeo/src/lib/st-table/st-table.component.html]: Remove directive from table

File: projects/egeo/src/lib/st-table/st-table.component.html (hunk 1 above)
Comment (jdiaz-stratio) [reply projects/egeo/src/lib/st-table/st-table.component.html]: Done

File: projects/egeo/src/lib/st-table/st-table.component.ts (hunk 2 above)
Comment (jdiaz-stratio) [reply projects/egeo/src/lib/st-table/st-table.component.ts]: Removed

File: projects/egeo/src/lib/st-table/st-table.component.html (hunk 3 above)
Comment (jdiaz-stratio) [reply projects/egeo/src/lib/st-table/st-table.component.html]: Removed

**Pull request closed without merging**